formulaire/src/main/java/fr/openent/formulaire/controllers/ResponseController.java

```diff
@@ -100,6 +100,39 @@ public void listMineByDistribution(HttpServerRequest request) {
         });
     }
 
+    @Post("/distributions/:distributionId/responses/multiple")
```

Comment:
doc API ?

Reply:
https://confluence.support-ent.fr/display/FOR/Response#operations-tag-default